src/cfml/system/services/ServerService.cfc

```diff
@@ -1208,6 +1207,10 @@ component accessors="true" singleton {
 			.append( '--cookie-secure' ).append( serverInfo.sessionCookieSecure )
 			.append( '--cookie-httponly' ).append( serverInfo.sessionCookieHTTPOnly );
 			
+			if( ConfigService.settingExists( 'preferedBrowser' ) && len(trim(ConfigService.getSetting( 'preferedBrowser' )))) {
```

Comment:
You can shorten this line to just
```
if( len( trim( ConfigService.settingExists( 'preferedBrowser', '' ) ) ) ) {
```
and don't forget to mind the Ortus spacing guidelines.  You need spaces between those parens.

Reply:
@mgmathus I copy/pasted the wrong method name above.  Should have been `getSetting()`
```js
if( len( trim( ConfigService.getSetting( 'preferedBrowser', '' ) ) ) ) {
```
There is no second parameter to the `settingExists()` method